lib/templates/deviceConfigGroup.json

```diff
@@ -0,0 +1,80 @@
+{
+  "title": "Service",
+  "description": "A service",
+  "additionalProperties": false,
+  "type": "object",
+  "properties": {
+    "configGroups": {
```

Comment:
I understand this file is *exactly* as lib/templates/deviceGroup.json except by this line, that in lib/templates/deviceGroup.json is "services" instead of "configGroups". Is my understanding correct?

I think is not a good idea to have two almost equal files for the same... the risk to forget one when doing some upgrade (for instance, a new field added to the payload) is high and soon they can get out of sync in repository.

Maybe a good approach is the following onoe:

1. Use just one file, that could be lib/templates/deviceConfigGroup.json but changed to use "configGroups" in this line (as it is the one we want in the future, and "service" is a kind of legacy)
2. In the places related with "configGroup" element, that template is used as is.
3. In the places related with "service" the template is changed on the fly before used. Something like this:

```
 const templateConfigGroup = require('../../templates/deviceConfigGroup.json');

let x = templateConfigGroup.properties.configGroups;
delete templateConfigGroup.properties.configGroups;
templateConfigGroup.properties = x;
```

This approach maybe could help also to unify and simplify processing code.

(don't use `x` as var name, but something more meaningful ;)

Reply:
Fixed in 424d59d